.trunk/plugins/trunk

```diff
@@ -0,0 +1 @@
+/Users/shivanshvij/.cache/trunk/plugins/https---github-com-trunk-io-plugins/v0.0.3
```

Comment:
This file should perhaps not be included in the repository

Reply:
Almost definitely